### DISCUSSION TITLE
Column name has number as its first character

### DISCUSSION THREAD

**Discussion by Sailaja-K-Manvitha** [nhibernate/nhibernate-core#3297] [Q&A]
Column name has number as its first character. how to declare in hbm file?

**Comment by hazzik** [comment]
Please provide some examples.

**Comment by Sailaja-K-Manvitha** [comment]
```xml
<property name = "fiftyyrs" column= "[50yrs]" type = "string" length = "800"/>
```

**Reply by fredericDelaporte** [reply]
So, even with SQL-Server specific identifier quoting, it fails? Have you tried the [NHibernate generic SQL identifier quoting](https://nhibernate.info/doc/nhibernate-reference/mapping.html#mapping-quotedidentifiers), the back-ticks?

```xml
<property name="fiftyyrs" column="`50yrs`" type="string" length="800"/>
```

(I have edit your answer which was appearing empty because the xml was not put into a code block.)
